wiki/Ranking_criteria/en.md

```diff
@@ -92,7 +93,7 @@ Important to understand before using:
   - `osu!` itself should never be used in the source field except in cases where the track was specifically created for osu! (also known as an [osu! original](/wiki/osu!_originals)), such as commissioned tournament tiebreaker tracks.
 - **Metadata that exceeds the field's limits (81 characters) must be shortened.** Start by dropping additional markers and if this still is not sufficient, indicate that the title has been shortened using `...` in a sensible place.
 - **If the artist or title fields were shortened to fit in field limits, the omitted information must be added to the tags.**
-- **If the song mapped is a licensed one in the [Featured Artist](/wiki/People/Featured_Artists) library, `featured artist` must be added to tags.**
+- **The `featured artist` tag must only be added to tracks listed in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists).** The tag is used to describe whether a track is licensed by osu!.
```

Comment:
the wording here no longer requires adding the featured artist tag. it needs to say more to the effect of "the tag is required if [...] and allowed only if [...]" (but theres probably a better way to word that)

Reply:
If that is an issue, it could be worded similar to what was originally proposed like:
```
The `featured artist` tag must be added only if the tracks used are listed in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists).** The tag is used to describe whether a track is licensed by osu!.
```

Moving the `only` ahead will make it clear that adding the tag is still a must, but only if that condition is met.

Will need to consult this with Wala however as they were the one to propose such wording.

Reply:
woops, didn't see this

```suggestion
- **The `featured artist` tag must be used for tracks listed in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists) and for such tracks only.** The tag is used to describe whether a track is licensed by osu!.
```

Reply:
Reads better on my end. Will wait for Clayton's OK before committing.

Reply:
yeah wala's suggestion is good